src/DAL/create_db.py

```diff
@@ -0,0 +1,31 @@
+from pymongo import MongoClient
+
+dbName = "panda-launder-db"
+products = "products"
+packages = "packages"
+providers = "providers"
+uri = "mongodb+srv://Guinness1759:gsn,h1759@guinness1759-lga2g.mongodb.net/test?retryWrites=true&w=majority"
```

Comment:
I think that `URI` should be passed from `env` variables.

this `URI` exposes `secrets`.

Reply:
Settings yaml file on it's way to this pull request